docs/classes/api.openapi.md

```diff
@@ -45,217 +42,229 @@
 
 \+ **new OpenAPI**(`__namedParameters`: { `handleToken`: *boolean* ; `key`: *string* ; `region`: *string* ; `schema`: *string* ; `secret`: *string*  }): [*OpenAPI*](api.openapi.md)
 
-#### Parameters:
+#### Parameters
 
-• **__namedParameters**: *object*
-
-Name | Type |
------- | ------ |
-`handleToken` | *boolean* |
-`key` | *string* |
-`region` | *string* |
-`schema` | *string* |
-`secret` | *string* |
+| Name | Type |
+| :------ | :------ |
+| `__namedParameters` | *object* |
+| `__namedParameters.handleToken` | *boolean* |
+| `__namedParameters.key` | *string* |
+| `__namedParameters.region` | *string* |
+| `__namedParameters.schema` | *string* |
+| `__namedParameters.secret` | *string* |
 
 **Returns:** [*OpenAPI*](api.openapi.md)
 
-Defined in: [api.ts:25](https://github.com/TuyaAPI/openapi/blob/fba2391/src/api.ts#L25)
+Defined in: [api.ts:25](https://github.com/imbenwolf/openapi/blob/eb338ed/src/api.ts#L25)
```

Comment:
Not sure why the generator changed this, can you take a look?

Reply:
I think it's because of the pre-commit hook (https://github.com/TuyaAPI/openapi/blob/master/package.json#L70). I guess it's great for development if in the same repository... For now i just commented it out so I can commit correctly but maybe we can optimize it in a future PR

Reply:
also travis runs `npm run document` but doesn't actually commit the generated docs so maybe you would have to quickly run the command on the master branch and commit them after the PR gets merged

Reply:
Yeah I'll run it manually, thanks.

Travis is kinda broken right now. 😛